frontend/ui/src/services/types/attachmentsTypes.ts

```diff
@@ -0,0 +1,45 @@
+import {attachmentsExtensions} from 'render';
+
+export const getAllSupportedAttachmentsForSource = (source: string) => {
+  if (source === 'twilio.whatsapp') source = 'twilioWhatsapp';
+
+  const imageFiles = attachmentsExtensions[source + 'ImageExtensions'];
+  const videoFiles = attachmentsExtensions[source + 'VideoExtensions'];
+  const audioFiles = attachmentsExtensions[source + 'AudioExtensions'];
+  const docsFiles = attachmentsExtensions[source + 'FileExtensions'];
+
+  const supportedDocsFiles = docsFiles ? docsFiles.join(', ') : '';
+  const supportedAudioFiles =
+    audioFiles && docsFiles ? audioFiles.join(', ') + ',' : audioFiles ? audioFiles.join(', ') : '';
+  const supportedVideoFiles =
+    videoFiles && audioFiles ? videoFiles.join(', ') + ',' : videoFiles ? videoFiles.join(', ') : '';
+  const supportedImageFiles =
+    imageFiles && videoFiles ? imageFiles.join(', ') + ',' : imageFiles ? imageFiles.join(', ') : '';
+
+  return `${supportedImageFiles} ${supportedVideoFiles} ${supportedAudioFiles} ${supportedDocsFiles}`;
+};
+
+export const getInputAcceptedFilesForSource = (source: string) => {
+  if (source === 'twilio.whatsapp') source = 'twilioWhatsapp';
+
+  const imageFiles = attachmentsExtensions[source + 'ImageExtensions'];
+  const videoFiles = attachmentsExtensions[source + 'VideoExtensions'];
+  const audioFiles = attachmentsExtensions[source + 'AudioExtensions'];
+  const docsFiles = attachmentsExtensions[source + 'FileExtensions'];
+
+  if (!imageFiles && !videoFiles && !audioFiles && !docsFiles) {
+    return null;
+  }
+
+  const supportedDocsFiles = docsFiles ? '.' + docsFiles.join(', .') : '';
+  const supportedAudioFiles =
+    audioFiles && docsFiles ? '.' + audioFiles.join(', .') + ',' : audioFiles ? audioFiles.join(', .') : '';
+  const supportedVideoFiles =
+    videoFiles && audioFiles ? '.' + videoFiles.join(', .') + ',' : videoFiles ? videoFiles.join(', .') : '';
+  const supportedImageFiles =
+    imageFiles && videoFiles ? imageFiles.join(', .') + ',' : imageFiles ? imageFiles.join(', .') : '';
+
+  const inputAcceptValue = `.${supportedImageFiles} ${supportedVideoFiles} ${supportedAudioFiles} ${supportedDocsFiles}`;
+
+  return inputAcceptValue;
+};
```

Comment:
added to `/Services` functions that dynamically fetch that supported files for each source